tests/template_tests/filter_tests/test_truncatewords_html.py

```diff
@@ -53,7 +53,7 @@ def test_truncate_complex(self):
             truncatewords_html(
                 "<i>Buenos d&iacute;as! &#x00bf;C&oacute;mo est&aacute;?</i>", 3
             ),
-            "<i>Buenos d&iacute;as! &#x00bf;C&oacute;mo …</i>",
+            "<i>Buenos días! ¿Cómo …</i>",
```

Comment:
Again, probably worth expanding the backward compatibility note I recommended above to explicitly mention the expansion of HTML entities. We've already agreed this is an acceptable trade-off and, in general, there should be little issues as long as everyone is using UTF-8 as they very much should be! _(If anyone is relying on anything else they'll have to write a custom filter to convert things back to entities as required. So be it.)_